### PR TITLE
[DELIVERABLE 2] SRS Document: Add Rationale + Priority and Changed Formatting for Usability Requirements

### PR DESCRIPTION
**Pull request by nivethakuruparan**
## Description

Added the rationale, and priority levels and also changed the reqs format for Usability Requirements.

## How Has This Been Tested?

Compiled locally

### CODE REVIEW

File: docs/SRS/SRS.tex [hunk 1]
```diff
@@ -530,59 +530,59 @@ \subsection{Style Requirements}
...
-  \item \textbf{Requirement:} The tool should communicate errors and issues politely and constructively.
-  \begin{itemize}[label={}]
-      \item \textbf{Fit Criteria:} User feedback should reflect that at least 80\% of users perceive error messages as helpful and courteous, rather than frustrating or vague.
-  \end{itemize}
-  \item \textbf{Requirement:} The tool should provide context-sensitive help that offers assistance based on the current user actions.
-  \begin{itemize}[label={}]
-      \item \textbf{Fit Criteria:} Help resources should be accessible within 1-3 clicks.
-  \end{itemize}
+\begin{enumerate}[label=UHR-UPL \arabic*., wide=0pt, leftmargin=*]
+    \item \emph{The tool shall provide context-sensitive help that offers assistance based on the current user actions.}\\
```
Comment (Sevhena) [inline docs/SRS/SRS.tex:567]: This is a duplicate from above (learning reqs)

**Review by Sevhena — CHANGES_REQUESTED** [review]
Looks good but you have a duplicate req

File: docs/SRS/SRS.tex (hunk 1 above)
Comment (nivethakuruparan) [reply docs/SRS/SRS.tex:567]: oop i didn't even notice that, thanks

**Review by Sevhena — APPROVED** [review]
LGTM

**Review by mmyaaaaa — APPROVED** [review]
LGTM

**Pull request merged**